docs/guides/how_to_guides/creating_batches/how_to_configure_a_runtime_data_connector.rst

```diff
@@ -0,0 +1,80 @@
+.. _how_to_guides__creating_batches__how_to_configure_a_runtime_data_connector:
+
+How to configure a RuntimeDataConnector
```

Comment:
Just a thought/problem to solve: We don't have a good way to highlight docs that only apply to the V3 API. Maybe we can start putting it in the title, e.g. How to configure a RuntimeDataConnector - V3 (Batch Request) API only

Reply:
Will do.